Scripts/Python/grsnWallPython.py

```diff
@@ -40,1207 +40,1187 @@
+                    SouthTubeClose.run(self.key)
+                    if(PtFindAvatar(events) != PtGetLocalAvatar()):
+                        SouthTubeExclude.clear(self.key)
+                elif(event[0] == kMultiStageEvent and event[1] == 0 and event[2] == kAdvanceNextStage):
+                    #Multistage complete
+                    if(PtFindAvatar(events) == PtGetLocalAvatar()):
+                        avatar = PtGetLocalAvatar()
+                        currentgender = avatar.avatar.getAvatarClothingGroup()
+                        if currentgender == kFemaleClothingGroup:
+                            clothing = FemaleSuit
+                        else:
+                            clothing = MaleSuit
+                        if not wornItem:
+                            wornItem = avatar.avatar.getAvatarClothingList()
+                            for item in wornItem:
+                                DefaultColor1.append(avatar.avatar.getTintClothingItem(item[0],1))
+                                DefaultColor2.append(avatar.avatar.getTintClothingItem(item[0],2))
+                        i = 1
+                        for item in clothing[0:]:
+                            avatar.avatar.netForce(1)
+                            avatar.avatar.wearClothingItem(item, 0)
+                            avatar.avatar.tintClothingItem(item, DefaultColor1[i], 0)
+                            avatar.avatar.tintClothingItemLayer(item, DefaultColor2[i], 2, 1)
+                            i += 1
+                        PtSendKIMessage(kDisableEntireYeeshaBook, 0)
+                        PtGetLocalAvatar().physics.warpObj(NorthTeamWarpPt.value.getKey())
+                        ageSDL["sWallPlayer"] = (-1,)
+                    if (ageSDL["sState"][0] == kEntry):
+                        self.ChangeGameState(kNorth, kGameInProgress)
+                        self.ChangeGameState(kSouth, kGameInProgress)
+                    SouthTubeOpen.run(self.key)
+            return
+        ### Win region ###
+        if(id == NorthTeamWin.id):
+            if(PtFindAvatar(events) == PtGetLocalAvatar()):
+                PtAtTimeCallback(self.key, 3.0, kLinkToNorthNexus)
+                if (ageSDL["grsnGrantMaintainerSuit"][0] and ageSDL["nState"][0] != kEnd and PtGetPlayerList()):
+                    avatar = PtGetLocalAvatar()
+                    currentgender = avatar.avatar.getAvatarClothingGroup()
+                    if currentgender == kFemaleClothingGroup:
+                        clothing = FemaleSuit
+                    else:
+                        clothing = MaleSuit
+                    for item in clothing[0:]:
+                        if (not self.IItemInCloset(avatar, item)):
+                            PtDebugPrint(f'DEBUG: grsnWallPython.OnNotify():  Adding {item} to your closet and wearing it.')
+                            avatar.avatar.addWardrobeClothingItem(item, ptColor().white(), ptColor().white())
```

Comment:
Should this be adding to the closet with the `white` color? I noticed you used the `DefaultColor1` and `DefaultColor2` for each piece in other places.

Reply:
Since the suit doesnt have any custom colors, adding to the closet with white is fine.